### ISSUE TITLE
Strange behavior with HVAC fan and aircoil + fan system

### ISSUE THREAD

**Issue by gforney** [firemodels/fds#1872]
```
Please complete the following lines...

FDS Version: 6.0 RC3
SVN Revision Number:14299
Compile Date: 12/20/2012
Smokeview Version/Revision:
Operating System: WIN7 (x64)

Describe details of the issue below:

I am simulation two rooms that share a common wall and a corridor. One room has an
equipment cabinet that generates heat via aircoil and volume_flow. the other room has
been emptied (to simplify the issue) and has just an HVAC fan that exhaust air out
of the empty room into the common corridor. there is a return "HOLE" that allows corridor
air to circulate into the empty room.

When the HVAC fan for the empty room is taken out of the model, the HVAC node temperature
for the heat generating cabinet show 1 behavior, and when the fan for the empty room
is in place, the node temperature show an wildly different behavior.

I am at a loss to understand how the fan in the empty room effects the node temperature
of the cabinet. The room with the cabinet has no leaks, windows or doors. 


```

Original issue reported on code.google.com by `louiea` on 2013-05-05 20:19:16

<hr>
- _Attachment: [RoomHeatLoad.fds](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-1907/comment-0/RoomHeatLoad.fds)_

**Comment by gforney** [comment]
```
It would be helpful if you simplified the case. One MESH, just two rooms. Just include
diagnostic output that demonstrates the problem.
```

Original issue reported on code.google.com by `mcgratta` on 2013-05-05 20:57:25

**Comment by gforney** [comment]
```
Also if you could post both versions of the input and if you could state the specific
 behavior you see.  
```

Original issue reported on code.google.com by `drjfloyd` on 2013-05-05 21:13:10

**Comment by gforney** [comment]
```
Attached are the input files with only a single mesh for the entire model. I believe
the room vent fan has a role in the problem, so I left corridor in. 

As you can see from the last column of the attached devc.csv files:

With the room vent fan: Temperature holds at 309K till about 36 seconds, then explodes
up to 340K. (the room fan is set to shut off at 25 seconds, which it does)

Without the room vent fan: temperature immediately jumps and fluctuates wildly at 340K.


The room fan isn't even in the same room as the equipment cabinet. 
```

Original issue reported on code.google.com by `louiea` on 2013-05-05 22:37:59

**Comment by gforney** [comment]
```
Looks like the attachments did not make it through...
```

Original issue reported on code.google.com by `louiea` on 2013-05-05 22:41:41

<hr>
- _Attachment: [RoomHeatLoad-With_Rm_Vent_Fan.fds](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-1907/comment-4/RoomHeatLoad-With_Rm_Vent_Fan.fds)_
- _Attachment: [RoomHeatLoad-No_Rm_Vent_Fan.fds](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-1907/comment-4/RoomHeatLoad-No_Rm_Vent_Fan.fds)_
- _Attachment: [RoomHeatLoad_devc-With-Rm_Vent_Fan.csv](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-1907/comment-4/RoomHeatLoad_devc-With-Rm_Vent_Fan.csv)_
- _Attachment: [RoomHeatLoad_devc-No-Rm_Vent_Fan.csv](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-1907/comment-4/RoomHeatLoad_devc-No-Rm_Vent_Fan.csv)_

**Comment by gforney** [comment]
```
This is not an HVAC problem.  The odd behavior you are seeing is due to the temperatures
at the inlet vent for the equipment rack.  I can get similar behavior to occur just
using fixed velocity inlet and outlets (i.e. without any &HVAC inputs).  

Kevin and Randy,

I have uploaded a simplified version of the case.  If you look at the temperature slice
shortly before the end of the simulation there is some form of instability forming
(you can see alternating hot and cold cells at the inlet vent at the bottom).




```

Original issue reported on code.google.com by `drjfloyd` on 2013-05-06 14:47:48

<hr>
- _Attachment: [RoomHeatLoad.fds](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-1907/comment-5/RoomHeatLoad.fds)_

**Comment by gforney** [comment]
```
fix cc
```

Original issue reported on code.google.com by `drjfloyd` on 2013-05-06 14:54:13

**Comment by gforney** [comment]
```
Interestingly, the attached works fine, where I have set TMPA to 20 C.  Setting this
case to TMPA=32.22 gives the problem.  Wonder if we are not correctly setting TMP_0(KK)
based on ambient temp somewhere.
```

Original issue reported on code.google.com by `randy.mcdermott` on 2013-05-06 19:25:55

<hr>
- _Attachment: [RoomHeatLoad_2.fds](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-1907/comment-7/RoomHeatLoad_2.fds)_

**Comment by gforney** [comment]
```
I tried Randy's test with different values of TMPA from 25 to 34. The issue begins to
manifest itself around TMPA=28 to TMPA=29. Perhaps it is a dT issue. 
```

Original issue reported on code.google.com by `louiea` on 2013-05-06 20:49:16

**Comment by gforney** [comment]
```
Second strange symptom is that things seem to work fine for DNS.
```

Original issue reported on code.google.com by `randy.mcdermott` on 2013-05-06 21:02:45

<hr>
- _Attachment: [RoomHeatLoad_2.fds](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-1907/comment-9/RoomHeatLoad_2.fds)_

**Comment by gforney** [comment]
```
Also works with CHECK_VN=T and VN_MAX=0.5 (on MISC), which is apparently the DNS setting
that fixed the problem.  So, I am ready to declare this to be a VN problem.

louiea, please add these parameters to your case and let me know if this works.
```

Original issue reported on code.google.com by `randy.mcdermott` on 2013-05-06 21:06:26

<hr>
- _Attachment: [RoomHeatLoad_2.fds](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-1907/comment-10/RoomHeatLoad_2.fds)_

**Comment by gforney** [comment]
```
It appears to have solved the problem in one of my test cases, the temperatures do no
go all over the place. I am testing my original case with the parameters now, takes
approx. 3 days to run.

The initial strange behavior now seems to make sense. When the other room's vent fan
was on, the high velocity it generated fixed the time step so that the VN constraint
was always in range. when the fan turned off, the time step got large enough that the
VN constraint was exceeded. 

Is this VN constraint a new constraint due to the improved LES turb. models? 
```

Original issue reported on code.google.com by `louiea` on 2013-05-06 22:51:43

**Comment by gforney** [comment]
```
Well, the Von Neumann stability constraint has always been around, but older versions
of the code used to ascribe instabilities to a variety of plausible causes.  Only now
that we have such a wealth of solid verification cases are we more confident that the
various sub-models are working correctly and that the basic numerical stability criteria
(CFL, VN) are probably to blame when things like this arise.  But near walls, where
the various wall models come into play, it gets even harder to isolate the issues.
 I spent a couple of hours working under the assumption we had a bug somewhere.

We have for now made the decision that CHECK_VN (which is not computationally "free
of charge") should not be default for LES.  But if enough of these issues crop up,
that may change.
```

Original issue reported on code.google.com by `randy.mcdermott` on 2013-05-07 01:02:25

**Comment by gforney** [comment]
```
Note that the VN_MIN and VN_MAX defaults listed in the user guide table and Section
6.4.9 are inconsistent.
```

Original issue reported on code.google.com by `koverholt` on 2013-05-07 06:43:08

**Comment by gforney** [comment]
```
Kris, thanks for the heads up.  I have corrected the code and user guide to make VN_MAX=0.5.
```

Original issue reported on code.google.com by `randy.mcdermott` on 2013-05-07 13:01:11

**Comment by gforney** [comment]
```
Thanks for looking at this, the problems in my simulations appear to be solved by adding
in the CHECK_VN=.TRUE. line.

Sorry for the trouble I caused by misdiagnosing the problem. 
```

Original issue reported on code.google.com by `louiea` on 2013-05-08 15:28:58

**Comment by gforney** [comment]
```
No trouble.  Thanks for letting us know that CHECK_VN=.TRUE. solved your problem.

```

Original issue reported on code.google.com by `drjfloyd` on 2013-05-08 15:35:10

**Comment by gforney** [comment]
```
One more comment: in the User Guide, the input table for MISC shows that CHECK_VN is
TRUE by default, but this is only true for DNS. And the CHECK_VN parameter is not discussed
in the text (i.e., near the VN_MIN and VN_MAX parameters in Section 6.4.9).
```

Original issue reported on code.google.com by `koverholt` on 2013-05-09 15:11:39

**Comment by gforney** [comment]
```
Thanks for the catch.  The guide has been corrected.
```

Original issue reported on code.google.com by `randy.mcdermott` on 2013-05-09 15:26:55

**Comment by gforney** [comment]
```
Since VN fixed was verified in #15 marking this as verfied
```

Original issue reported on code.google.com by `drjfloyd` on 2013-07-23 13:02:07
